_editorial_policies/02_editor_instructions.md

```diff
@@ -11,23 +11,26 @@ This provides instructions to editors handling manuscripts for LiveCoMS.
 
 ## Workflow overview
 
-Authors contact LiveCoMS (specifically, the Lead Editor in a particular subject area) with a presubmission letter, proposing an article in a particular area.
-Once this is approved, they proceed with article preparation and ultimately submit it to that section of LiveCoMS.
-The Lead Editor then checks that the article is consistent with what was proposed in the presubmission letter, and then assigns it to an appropriate Associate Editor who manages the review process and makes a final decision on the manuscript, reporting the final decision to the authors.
+Authors contact LiveCoMS (specifically, the Section Lead Editor in that topic) with a presubmission letter, proposing an article in a particular area. The Section Lead Editor assigns this presubmission letter to and Associate Editor, who works with the authors to revise the article plans to be suitable for LivCoMS, and the Section Lead Editor signs off on the final plans.
+Once the presubmission letter is approved, the author proceeds with article preparation and ultimately submit it to that section of LiveCoMS.
+The Lead Editor then checks that the article is consistent with what was proposed in the presubmission letter, and then assigns it to an appropriate Associate Editor who manages the review process and makes a final decision on the manuscript, reporting the final decision to the authors. Usually this Associate Editor will be the same one that handled the original presubmission letters.
+
+## Pre-submission letters
+
+Assuming the subject matter proposed in the is within the scope of the LiveCoMS and is otherwise acceptable, the Section Lead Editor assigns this presubmission letter to and Associate Editor. Reasons for the letter not being acceptable include ignoring key required elements of the presubmission letter, being too similar to existing LiveCoMS articles. The Section Lead Editor signs off on the final presubmission letter, and the Associate Editor notifies the author that the letter is approved.  Instrutions for the presubmission letter are [found here](https://github.com/livecomsjournal/livecomsjournal.github.io). 
```

Comment:
This paragraph says the section lead handles the presubmission letter but the paragraph before says it's the associate editor who does. :)

Reply:
I am trying to say that the iterations of the presubmission letter are handled by the associate editor, but they should still check with the section lead before final approval.  The basic idea is if there are any issues about scope, etc,. we want to catch them earlier.

Reply:
Since the author instructions indicate letter goes to section lead, we could clarify here:
"the Section Lead Editor assigns this presubmission letter to and Associate Editor WHO HANDLES IT FROM THAT POINT FORWARD."

Also, this sentence should be changed: "The Section Lead Editor signs off on the final presubmission letter, and the Associate Editor notifies the author that the letter is approved."
We probably want to reverse the roles here, if I understand correctly the new policy.